pydevices/acq400Devices/acq400_base.py

```diff
@@ -119,6 +159,17 @@ def arm(self):
         thread.start()
     ARM=arm
 
+    def trig(self, msg=''):
+        message = str(msg)
+        uut = acq400_hapi.Acq400(self.node.data())
+        self.dprint(1, "The trigger message for streamming is: %s", message)
+
+        wrtdtx = '1 --tx_id=' + message
```

Comment:
We should make sure `message` is escaped, or otherwise won't break when we add it to that command

Reply:
Now the message string is escaped.